Diagnostic/portal.xml.template

```diff
@@ -1,93 +1,96 @@
-        <Unpivot columnName="CounterName" columnValue="Value" columns="AvailableMemory, PercentAvailableMemory, UsedMemory,PercentUsedMemory, PercentUsedByCache, PagesPerSec, PagesReadPerSec, PagesWrittenPerSec, AvailableSwap, PercentAvailableSwap, UsedSwap, PercentUsedSwap">
-      <MapName name="AvailableMemory" scaleUp="1048576">\Memory\AvailableMemory</MapName>
-<MapName name="PercentAvailableMemory">\Memory\PercentAvailableMemory</MapName>
-<MapName name="UsedMemory" scaleUp="1048576">\Memory\UsedMemory</MapName>
-<MapName name="PercentUsedMemory">\Memory\PercentUsedMemory</MapName>
-<MapName name="PercentUsedByCache">\Memory\PercentUsedByCache</MapName>
-<MapName name="PagesPerSec">\Memory\PagesPerSec</MapName>
-<MapName name="PagesReadPerSec">\Memory\PagesReadPerSec</MapName>
-<MapName name="PagesWrittenPerSec">\Memory\PagesWrittenPerSec</MapName>
-<MapName name="AvailableSwap" scaleUp="1048576">\Memory\AvailableSwap</MapName>
-<MapName name="PercentAvailableSwap">\Memory\PercentAvailableSwap</MapName>
-<MapName name="UsedSwap" scaleUp="1048576">\Memory\UsedSwap</MapName>
-<MapName name="PercentUsedSwap">\Memory\PercentUsedSwap</MapName>
-
-    </Unpivot> </OMIQuery>
-
-     <OMIQuery cqlQuery="SELECT PercentIdleTime,PercentUserTime,PercentNiceTime,PercentPrivilegedTime,PercentInterruptTime,PercentDPCTime,PercentProcessorTime,PercentIOWaitTime from SCX_ProcessorStatisticalInformation where Name=&apos;_Total&apos;" eventName="cpuStats" omiNamespace="root/scx" sampleRateInSeconds="15" storeType="local">
-       <Unpivot columnName="CounterName" columnValue="Value" columns=" PercentUserTime,PercentNiceTime,PercentPrivilegedTime,PercentInterruptTime,PercentDPCTime,PercentProcessorTime,PercentIOWaitTime">
-       <MapName name="PercentIdleTime">\Processor\PercentIdleTime</MapName>
-<MapName name="PercentUserTime">\Processor\PercentUserTime</MapName>
-<MapName name="PercentNiceTime">\Processor\PercentNiceTime</MapName>
-<MapName name="PercentPrivilegedTime">\Processor\PercentPrivilegedTime</MapName>
-<MapName name="PercentInterruptTime">\Processor\PercentInterruptTime</MapName>
-<MapName name="PercentDPCTime">\Processor\PercentDPCTime</MapName>
-<MapName name="PercentProcessorTime">\Processor\PercentProcessorTime</MapName>
-<MapName name="PercentIOWaitTime">\Processor\PercentIOWaitTime</MapName>
-
-         </Unpivot>
+      <Unpivot columnName="CounterName" columnValue="Value" columns="AvailableMemory, PercentAvailableMemory, UsedMemory,PercentUsedMemory, PercentUsedByCache, PagesPerSec, PagesReadPerSec, PagesWrittenPerSec, AvailableSwap, PercentAvailableSwap, UsedSwap, PercentUsedSwap">
```

Comment:
the web shows a lot of change on portal.xml.template. but when I compare some of the changed, highlighted texts, it seems there is no real word change. it seems to have indentation (tab vs space) changes.  I recommend this to be changed back to its original space so that it is clear what real text is changed.